aioauth_fastapi/router.py

```diff
@@ -87,7 +104,9 @@ async def token_introspect(request: Request, form: TokenIntrospectForm = Depends
 
     @router.get("/authorize")
     async def authorize(request: Request, query: Query = Depends()):
```

Comment:
This use of `aioauth.requests.Query` here is surprising, can you sanity check this while you're in the neighborhood? I'm happy to remove it if isn't needed.

Reply:
The tests pass if this `query` arg is removed and if a `form` arg elsewhere in the file is removed. Not sure that means it can be removed though.

Reply:
Checked locally. I think `query` can be removed.